react_app/src/components/LoginPage/LoginPageBox.js

```diff
@@ -13,7 +13,7 @@ const useStyles = makeStyles({
 
 const LoginPageBox = (props) => {
     const classes = useStyles();
-
+    
     return (
```

Comment:
Looks like no changes were made to this file?? Pls revert the changes.  Thanks.

Reply:
Yeah, we don't currently need to make changes to this file for the purposes of this PR.